src/fides/api/ops/api/v1/endpoints/policy_endpoints.py

```diff
@@ -297,6 +366,82 @@ def delete_rule(
     rule.delete(db=db)
 
 
+def get_rule_target_or_error(
+    db: Session,
+    policy_key: FidesOpsKey,
+    rule_key: FidesOpsKey,
+    rule_target_key: FidesOpsKey,
+) -> RuleTarget:
+    """
+    Helper method to load Rule Target or throw a 404.
+    Also throws a 404 if a `Policy` or `Rule` with the given keys can't be found.
+    """
+    logger.info("Finding rule target with key '{}'", rule_target_key)
+    rule: Rule = get_rule_or_error(db, policy_key, rule_key)
+    rule_target = RuleTarget.filter(
+        db=db,
+        conditions=(
+            (RuleTarget.rule_id == rule.id) & (RuleTarget.key == rule_target_key)
+        ),
+    ).first()
+    if not rule_target:
+        raise HTTPException(
+            status_code=HTTP_404_NOT_FOUND,
+            detail=f"No Rule Target found for key {rule_target_key}.",
+        )
+
+    return rule_target
+
+
+@router.get(
+    urls.RULE_TARGET_LIST,
+    status_code=HTTP_200_OK,
+    response_model=Page[schemas.RuleTarget],
+    dependencies=[Security(verify_oauth_client, scopes=[scope_registry.RULE_READ])],
```

Comment:
Note to self: existing target endpoints also use the rule read scope, there's no separate scope for targets